WordPress/src/main/res/values/strings.xml

```diff
@@ -986,6 +986,30 @@
     <string name="editor_post_settings_set_featured_image">Set Featured Image</string>
 
 
+    <!-- Aztec -->
+    <string name="dialog_title">Insert link</string>
```

Comment:
Just noticed this, but we should probably give it a more descriptive name (both here and in the editor lib) - maybe `insert_link_dialog_title`.